packages/catalog-generator/pom.xml

```diff
@@ -82,6 +77,12 @@
       <artifactId>commons-cli</artifactId>
       <version>1.9.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.camel</groupId>
+      <artifactId>camel-yaml-dsl</artifactId>
+      <version>${version.camel}</version>
+      <scope>test</scope>
```

Comment:
Without having `<scope>test</scope>`, this forces the runtime to load the latest Camel YAML DSL schema, making the catalog generation fail.